frontend/public/locales/en/public.json

```diff
@@ -526,6 +526,7 @@
   "Successfully updated the environment variables.": "Successfully updated the environment variables.",
   "Environment variables set from parent": "Environment variables set from parent",
   "View environment for resource": "View environment for resource",
+  "owners:": "owners:",
```

Comment:
Hmm, it appears that the   `"owners:": "owners:",` was missing previously. Wondering how the parser and CI i18n test could skip that previously. That's a good catch if it was the source of the issue.